### PR TITLE
ci: shellcheck integration tests

### PR DESCRIPTION
**Pull request by srenatus**
I was waiting for a thing to finish, when it came over me.

I know some of these might be controversial, and the effort might outweight the benefit -- but I'd still prefer we shellcheck'ed all the things. 😉

### CODE REVIEW

File: integration/tests/deep_upgrade.sh [hunk 1]
```diff
@@ -26,24 +29,32 @@ do_create_config() {
 }
 
 do_deploy() {
-    cat $DEEP_UPGRADE_PATH > $test_manifest_path
```
Comment (stevendanna) [inline integration/tests/deep_upgrade.sh]: 🤣 Unsure why I did it this way

File: .studiorc [hunk 2]
```diff
@@ -41,12 +41,12 @@ export ELASTICSEARCH_URL="http://localhost:10141"
 export POSTGRESQL_URL="localhost:5432"
 # Ingest
 export INGEST_URL="https://localhost:10122"
-export INGEST_PORT=$(echo $INGEST_URL | awk -F: '{ print $NF }')
-export INGEST_HOST=$(echo $INGEST_URL | awk -F: '{ print $2 }' | tr -d '/')
+export INGEST_PORT="localhost"
```
Comment (srenatus) [inline .studiorc]: I've found this a bit silly, so I've ended it 😄

Comment (stevendanna) [reply .studiorc]: Uh oh, are these reversed?  Should this be 10122 and the next one "localhost"?

Comment (srenatus) [reply .studiorc]: 😅 oh no!

I'll fix that.

Comment (srenatus) [reply .studiorc]: https://github.com/chef/automate/pull/933

**Pull request merged**